### PR TITLE
Add text on GPU compute model

### PR DESCRIPTION
**Pull request by atgutier**

### CODE REVIEW

File: author-list.tex [hunk 1]
```diff
@@ -1,5 +1,5 @@
...
...ico Amslinger, Matteo Andreozzi, Adri\`a Armejach, Nils Asmussen, Brad Beckmann, Srikant Bharadwaj, Gabe Black, Gedare Bloom, Bobby R. Bruce, Daniel Rodrigues Carvalho, Jeronimo Castrillon, Lizhong Chen, Nicolas Derumigny, Stephan Diestelhorst, Wendy Elsasser, Carlos Escuin, Marjan Fariborz, Amin Farmahini-Farahani, Pouya Fotouhi, Ryan Gambord, Jayneel Gandhi, Dibakar Gope, Thomas Grass, Anthony Gutierrez, Bagus Hanindhito, Andreas Hansson, Swapnil Haria, Austin Harris, Timothy Hayes, Adrian Herrera, Matthew Horsnell, Syed Ali Raza Jafri, Radhika Jagtap, Hanhwi Jang, Reiley Jeyapaul, Timothy M. Jones, Matthias Jung, Subash Kannoth, Hamidreza Khaleghzadeh, Yuetsu Kodama, Tushar Krishna, Tommaso Marinelli, Christian Menard, Andrea Mondelli, Miquel Moreto, Tiago M\"uck, Omar Naji, Krishnendra Nathella, Hoa Nguyen, Nikos Nikoleris, Lena E. Olson, Marc Orr, Binh Pham, Pablo Prieto, Trivikram Reddy, Alec Roelke, Mahyar Samani, Andreas Sandberg, Javier Setoain, Boris Shingarov, Matthew D. Sinclair, Tuan Ta, Rahul Thakur, Giacomo Travaglini, Michael Upton, Nilay Vaish, Ilias Vougioukas, William Wang, Zhengrong Wang, Norbert Wehn, Christian Weis, David A. Wood, Hongil Yoon, \'Eder F. Zulian
```
Comment (mattsinc) [inline author-list.tex:2]: I think Brad normally uses "Bradford M. Beckmann" in papers, no?

File: changes/gpu-compute-model.tex [hunk 2]
```diff
@@ -1,8 +1,93 @@
...
+At a high level the CU pipeline modeled in gem5 consists of four stages: \textit{fetch},
+\textit{scoreboard check}, \textit{schedule}, and \textit{execute}. The fetch stage
+is responsible for arbitrating instruction fetch, storing fetched data,
+and decoding instructions for all wavefronts on the CU. The scoreboard check stage is
+responsible for ensuring that intra-wavefront dependences are
+correctly handled before instructions can be scheduled. The schedule stage is
+responsible for arbitrating among the wavefronts presented by
+the scoreboard check stage for each execution unit type. Once an instruction is dispatched
+to an execution unit it is guaranteed to execute.
+This "execute at execute" approach is the same as used by the gem5 in-order and
```
Comment (powerjg) [inline changes/gpu-compute-model.tex]:
```suggestion
This ``execute at execute'' approach is the same as used by the gem5 in-order and
```

As it is, this won't render correctly when compiled.

File: changes/gpu-compute-model.tex [hunk 3]
```diff
@@ -1,8 +1,93 @@
...
+O3 cpu models. The execute stage executes the instruction by calling the \textit{execute}
```
Comment (powerjg) [inline changes/gpu-compute-model.tex]:
```suggestion
out-of-order CPU models. The execute stage executes the instruction by calling the \textit{execute}
```

File: main.tex [hunk 4]
```diff
@@ -73,6 +73,12 @@ \section{Acknowledgements}
...
...agen, Weiping Liao, Pin-Yen Lin, Nicholas Lindsay, Yifei Liu, Gabe Loh, Andrew Lukefahr, Palle Lyckegaard, Jiuyue Ma, Xiaoyu Ma, Andriani Mappoura, Jose Marinho, Bertrand Marquis, Maxime Martinasso, Sean McGoogan, Mingyuan, Monir Mozumder, Malek Musleh, Earl Ou, Xin Ouyang, Rutuja Oza, Andrea Pellegrini, Arthur Perais, Adrien Pesle, Polydoros Petrakis, Anis Peysieux, Christoph Pfister, Sujay Phadke, Ivan Pizarro, Matthew Poremba, Brandon Potter, Siddhesh Poyarekar, Nathanael Premillieu, Sooraj Puthoor, Jing Qu, Prakash Ramrakhyani, Steve Reinhardt, Isaac Richter, Paul Rosenfeld, Shawn Rosti, Ali Saidi, Karthik Sangaiah, Ciro Santilli, Robert Scheffel, Sophiane Senni, Korey Sewell, Faissal Sleiman, Maximilian Stein, Po-Hao Su, Chander Sudanthi, Kanishk Sugand, Dam Sunwoo, Koan-Sin Tan, Michiel Van Tol, Erik Tomusk, Christopher Torng, Ashkan Tousi, Sergei Trofimov, Avishai Tvila, Ani Udipi, Jordi Vaquero, Llu\'is Vilanova, Wade Walker, Yu-hsin Wang, Bradley Wang, William Wang, Moyang Wang, Zicong Wang, Vince Weaver, Uri Wiener, Sean Wilson, Severin Wischmann, Willy Wolff, Yuan Yao, Jieming Yin, Bjoern A. Zeeb, Dongxue Zhang, Tao Zhang, Xianwei Zhang, Zhang Zheng, Chuan Zhu, Chen Zou.
```
Comment (powerjg) [inline main.tex]: Any authors added to the author list should be removed from here. I see that Tony is still in this list. I'm not sure who else was added to the author list.

**Review by powerjg — COMMENTED** [review]
A couple of small things. I can take care of them, if that's easier for you.

File: changes/gpu-compute-model.tex [hunk 5]
```diff
@@ -1,8 +1,93 @@
 \subsection[GPU Compute Model]{GPU Compute Model\footnote{by Anthony Gutierrez}}
 \label{sec:gpu}
 
-GPUs have become an important part of the system design for high-performance computing, machine learning, and many other workloads.
-Thus, we have integrated a compute-based GPU model into gem5~\cite{GutierrezBeckmann2018-amdAPU}.
+GPUs have become an important part of the system design for high-performance
+computing, machine learning, and many other workloads. Thus, we have integrated
+a compute-based GPU model into gem5~\cite{GutierrezBeckmann2018-amdAPU}. The
+GPU model is based on AMD's Graphics Core Next (GCN) architecture
+\footnote{http://amd.com/en/technologies/gcn}, and at the time of this writing
```
Comment (mattsinc) [inline changes/gpu-compute-model.tex]:
```suggestion
- at the time of this writing
+ currently
```

(I can also submit a separate pull request for this if it requires you going through legal again)

**Review by mattsinc — COMMENTED** [review]
Same as Jason, I can make these changes myself it's a problem.

File: author-list.tex (hunk 1 above)
Comment (atgutier) [reply author-list.tex:2]: I thought that too, but he's asked me on recent papers we've published to use "Brad Beckmann".

File: changes/gpu-compute-model.tex (hunk 5 above)
Comment (mattsinc) [reply changes/gpu-compute-model.tex]: Sorry, the suggested edit did not show up properly.  It was just replacing "at the time of this writing" with "currently", not removing the footnote.

File: changes/gpu-compute-model.tex (hunk 2 above)
Comment (atgutier) [reply changes/gpu-compute-model.tex]: Done

File: changes/gpu-compute-model.tex (hunk 3 above)
Comment (atgutier) [reply changes/gpu-compute-model.tex]: Done

File: main.tex (hunk 4 above)
Comment (atgutier) [reply main.tex]: Done. I removed Brad and added him as an author too.

File: changes/gpu-compute-model.tex (hunk 5 above)
Comment (atgutier) [reply changes/gpu-compute-model.tex]: It's fine, I understood. Done.

**Comment by powerjg** [comment]
Thanks, Tony!

**Pull request merged**